CCL.Creator/Validators/CarValidators.cs

```diff
@@ -352,9 +352,17 @@ private static IEnumerable<Result> CheckModelVariant(GameObject model)
             // bounding collider
             var collision = collidersRoot.FindSafe(CarPartNames.COLLISION_ROOT);
             var collisionComp = collision ? collision!.GetComponent<BoxCollider>() : null;
+
+            if (!collisionComp)
+            {
+                // If the root does not have a collider itself, accept it anyways if any of its
+                // children have one.
+                collisionComp = collision ? collision!.GetComponentInChildren<BoxCollider>() : null;
+            }
```

Comment:
We can actually condense this to a single line I believe, since GetComponentInChildren also searches the level that it's called on.

```csharp
var collisionComp = collision ? collision!.GetComponentInChildren<Collider>() : null;
```

Reply:
Just read the docs, seems like that's the case.
And I've been doing the 2 checks this whole time everywhere...